ui/pages/home/index.scss

```diff
@@ -176,4 +176,35 @@
       margin-left: 44px;
     }
   }
+
+  &__new-tokens-imported-notification {
+    margin-bottom: 4px;
+    margin-left: 8px;
```

Comment:
Have you tried looking at these changes in `RTL` direction?  It may be the case that using `margin-inline-start` would make positioning look good in both RTL and LTR.